ci/templatetags/settings_export.py

```diff
@@ -29,7 +29,12 @@ def installed_gitservers(request):
                 "hostname": s["hostname"],
                 "icon_class": s["icon_class"],
                 "html_url": s["html_url"],
+                "displayname" : ""
                 }
+        if "hpcgitlab.hpc.inl.gov" in s["hostname"]:
+            d["displayname"] = "HPC"
+        elif "gitlab.software.inl.gov" in s["hostname"]:
+            d["displayname"] = "External"
```

Comment:
add the config file read method as discussed (just saying as a reminder)!
```python
if "login_label" in s.keys():
    d["displayname"] = s["login_label"]
```
(or whatever you decide to key the label as)

Reply:
It would probably be better to set these custom display names in `civet/settings.py` which is where each git server is configured. Each git server just has a dictionary for settings so it would be easy to add `displayname` there. On the off chance anybody outside of INL is using civet, they would probably have different names which they could easily change in the git server config without modifying a random source file.

Reply:
My thoughts exactly :)

Good to hear from ya!!